stdlib/Makefile

```
@@ -264,7 +264,7 @@ depend:
 	  > .depend.tmp
 	$(CAMLDEP) $(DEPFLAGS) -pp "$(AWK) -f ./remove_module_aliases.awk" \
 	  stdlib.ml stdlib.mli >> .depend.tmp
-	$(strings $(which sed) | grep -q 'GNU') || (echo "the sed in PATH is not GNU sed" && exit 1)
+	$$(grep -q 'GNU' "$(which sed)") || {echo "the sed in PATH is not GNU sed" && exit 1}
```

```suggestion
	$$(grep -q 'GNU' "$$(which sed)") || { echo "the sed in PATH is not GNU sed" && exit 1 }
```

My bad I forgot a $ (feel free to squash)